metadata.go

```diff
@@ -80,12 +82,36 @@ var debianLicenseText = map[string]string{
 `,
 }
 
+var githubRegexp = regexp.MustCompile(`github\.com/([\w._-]+/[\w._-]+)`)
+
+// regex match github.com/([^/]+/[^/]+) on reply of https://gopkg/?go-get=1
+func findGitHubRepo(gopkg string) (string, error) {
+	if strings.HasPrefix(gopkg, "github.com/") {
+		return gopkg[len("github.com/"):], nil
+	}
+	resp, err := http.Get("https://" + gopkg + "?go-get=1")
+	if err != nil {
+		return "", err
+	}
+	body, err := ioutil.ReadAll(resp.Body)
+	resp.Body.Close()
+	if err != nil {
+		return "", err
+	}
+	repoMatch := githubRegexp.FindSubmatch(body)
```

Comment:
Instead of just applying a regexp, I think it’d be more robust/cleaner to actually extract the meta tags. See https://godoc.org/golang.org/x/net/html#example-Parse for an example of how to do that.